src/pages/account/[address].tsx

```diff
@@ -0,0 +1,109 @@
+import { Container, Divider, VStack, Box, Link } from '@chakra-ui/react';
+import { Trans, t } from '@lingui/macro';
+import { graphql, Link as GatsbyLink, withPrefix } from 'gatsby';
+import type { FunctionComponent } from 'react';
+import { getAddress } from 'viem';
+
+import banner from '../../assets/images/seo/home.png';
+import {
+  AccountProfileBanner,
+  AccountProfileTabs,
+  AccountInfo,
+} from '../../features/account';
+import type { Fields } from '../../utils';
+import NotFound from '../404';
+
+type AccountPageProps = {
+  params: {
+    address: string;
+  };
+};
+
+const AccountPage: FunctionComponent<AccountPageProps> = ({ params }) => {
+  let { address } = params;
+  try {
+    address = getAddress(params.address);
+  } catch {
+    return (
+      <Box data-testid="account-profile-not-found">
+        <NotFound />
+      </Box>
+    );
+  }
```

Comment:
I would move this to a separate function.

Reply:
Can you advice us?

Reply:
```ts
function parseAddress(address: Hex): Hex | null {
  try {
    return getAddress(address);
  } catch {
    return null;
  }
}

// ...
const address = parseAddress(params.address);
if (!address) {
  return 404;
}
```